browser/index.js

```diff
@@ -1,22 +1,20 @@
-var _algos = require('./algos')
 var createHash = require('create-hash')
+var stream = require('stream')
 var inherits = require('inherits')
 var sign = require('./sign')
-var stream = require('stream')
 var verify = require('./verify')
 
-var algos = {}
-Object.keys(_algos).forEach(function (key) {
-  algos[key] = algos[key.toLowerCase()] = _algos[key]
+var algorithms = require('./algorithms.json')
+Object.keys(algorithms).forEach(function (key) {
+  algorithms[key].id = new Buffer(algorithms[key].id, 'hex')
```

Comment:
because later it was initialized in [algos.js](https://github.com/crypto-browserify/browserify-sign/blob/4faa5baa5c2fd816232c286046b571f92db01207/browser/algos.js#L5)